src/pages/workspace/workflows/WorkspaceWorkflowsPage.tsx

```diff
@@ -78,7 +83,7 @@ function WorkspaceWorkflowsPage({policy, betas, route}: WorkspaceWorkflowsPagePr
                                   hoverAndPressStyle={[styles.mr0, styles.br2]}
                               />
                           ),
-                          isActive: policy?.harvesting?.enabled ?? false,
+                          isActive: (policy?.harvesting?.enabled && policy.autoReportingFrequency !== CONST.POLICY.AUTO_REPORTING_FREQUENCIES.INSTANT) ?? false,
```

Comment:
FYI we MIGHT want to use `policyUtils.isInstantSubmitEnabled` here? This soon will also check that `policy?.autoReporting` is set to `true` -> Because if `policy?.autoReporting` is `false`, we don't want to consider that policy "on instant submit"

Reply:
cc @lakchote